src/ckeditor.js

```diff
@@ -12,9 +14,56 @@
  * @singleton
  */
 
-CKEDITOR.define( function() {
+CKEDITOR.define( [ 'editor', 'mvc/collection', 'promise' ], function( Editor, Collection, Promise ) {
 	var CKEDITOR = {
 		/**
+		 * A collection containing all editor instances created.
+		 */
+		instances: new Collection(),
+
+		/**
+		 * Creates an editor instance for the provided DOM element.
+		 *
+		 * The creation of editor instances is an asynchronous operation, therefore a promise is returned by this
+		 * method.
+		 *
+		 *     CKEDITOR.create( '#content' );
+		 *
+		 *     CKEDITOR.create( '#content' ).then( function( editor ) {
+		 *         // Manipulate "editor" here.
+		 *     } );
+		 *
+		 * @param {String|Object} element An element selector or a DOM element, which will be the source for the
+		 * created instance.
+		 * @returns {Promise} A promise, which will be fulfilled with the created editor.
+		 */
+		create: function( element ) {
+			var that = this;
+
+			return new Promise( function( resolve, reject ) {
+				// If a query selector has been passed, transform it into a real element.
+				if ( typeof element == 'string' ) {
+					element = document.querySelector( element );
```

Comment:
IMO - KISS. Let's have the `CKEDITOR.create()` do one thing - create an editor based on DOM element. We need this method anyway, because the more complex scenarios should be based on a basic method like this.

Then, we can think how to initialise many editors. For instance that could be done by:

```
CKEDITOR.find( '.editor' ).forEach( CKEDITOR.create );
```

Or if you have jQ:

```
$( '.editor' ).each( CKEDITOR.create );
```

Or if you have nothing and there's no `CKEDITOR.find`:

```
[].slice.call( ( document.querySelectorAll( '.editor' ) ) ).forEach( CKEDITOR.create );
```

So the only think I would assure is that the `create()` method has the context bound.